Cineme.h

```diff
@@ -0,0 +1,116 @@
+
+#include "Movie.h"
+
+
+class Cinema
+{
+private:
+    vector<Movie> movies;
+    string name;
+    
+    int seats;
+
+public:
+    void printCinema()
+    {
+        cout << "Cinema: " << name << ";" << endl
+             << endl;
+    }
+    Cinema(string tmpName, int tmpSeats)
+    {
+        name = tmpName;
+
+        seats = tmpSeats;
+    }
+
+    void addNewMovieToCinema(Movie movie)
+    {
+        movies.push_back(movie);
+    }
+
+    void filterMovieByGenre(type genreFilter)
+    {
+        for (int i = 0; i < movies.size(); i++)
+        {
+            if (movies.at(i).getGenre() == genreFilter)
+            {
+                movies.at(i).print();
+            }
+        }
+    }
+    
+    void printAllMovies()
+    {
+        for (int i = 0; i < movies.size(); i++)
+        {
+            movies.at(i).print();
+        }
+
+        cout << "Count of movies : " << movies.size() << endl;
+    }
+
+    void sortMoviesByRank()
```

Comment:
The name of method shoud be in uppercase.

Reply:
Can be as it is.